spec/workers/search/remove_from_index_worker_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe Search::RemoveFromIndexWorker, type: :worker do
+  let(:worker) { subject }
+
+  include_examples "#enqueues_on_correct_queue", "medium_priority", ["searchables_#{Rails.env}", "users-456"]
```

Comment:
Is there a specific reason to use `Rails.env` here? Won't it always be `test` given that this is a spec?

Reply:
I thought the same thing - I could go either way on this one.

My thinking for keeping it as `Rails.env` is when we re-visit this spec in the future, it's easier to understand right away that the index is actually named based on the environment when the worker is called. So if someone were debugging it would be easier to see that they could check various indices on Algolia with the proper name.

Re-visiting this file in the future without all the context fresh in mind, `"searchables_test"` might read as some throwaway String and is less obvious (at least at first) that those indices are named by the environment.

Reply:
Got it. That's a valid point. Do you maybe want to pull it out into `let` and call it `index_name` or something? At least this way we only have to change it once if we ever change our mind on this.

Reply:
I went to do this and I realized it doesn't quite work because we can't pass `let` variables into shared examples. In the past, I've fixed this by tweaking the shared example to account for that. But since this example is very generic I didn't see a great way to handle that. And then if we updated it to a `let` variable, the specs below could use it but the shared example would still have to be the "hardcoded" String which feels less than ideal. In that scenario, I could see a situation where someone might quickly change the `let` variable in the future and not realize the shared example isn't using that same variable.

What do you think, any thoughts on how to get that `let` variable in the shared example? Should we go ahead and just use `let` and the "hardcoded" String?

Reply:
My main thought is that you probably should just ignore my OCD and leave as is 😉 We definitely reached the limit of amount of time this comment was worth. Thanks for exploring the option though!